docs/developer/dashboard.md

```diff
@@ -68,6 +68,16 @@ yarn run start
 
 As an alternative to using [Telepresence](https://www.telepresence.io/) you can use the default [Create React App API proxy](https://create-react-app.dev/docs/proxying-api-requests-in-development/) functionality.
 
+First add the desired host:port to the package.json:
+
+```patch
+-  }
++  },
++  "proxy": "http://127.0.0.1:8080"
+```
+
+> **NOTE**: The [proxy](../../dashboard/package.json#L176) `key:value` has already added to the `package.json` for convenience but you can change the `host:port` values to meet your needs.
```

Comment:
This note is no longer correct, I think?

Reply:
right, forgot to update the comment